data.sql

```diff
@@ -121,4 +121,245 @@ WHERE name = 'Charmander'
 UPDATE animals
 SET owners_id = 5
 WHERE name = 'Angemon'
-  OR name = 'Boarmon';
\ No newline at end of file
+  OR name = 'Boarmon';
+--vets/specializations
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES('William Tatcher', 45, '2000-04-23');
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES('Maisy Smith', 26, '2019-01-17');
+INSERT INTO vets(name, age, date_of_graduation)
+VALUES('Stephanie Mendez', 64, '1981-04-05');
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES ('Jack Harkness', 38, '2008-08-06');
+INSERT INTO specializations (species_id, vets_id)
+VALUES (1, 1),
+  (1, 3),
+  (2, 3),
+  (2, 4);
+-- animal visits
+INSERT INTO visits (
+    animal_id,
+    vet_id,
+    date_of_visit
+  )
+VALUES (
+    (
+      SELECT id
+      FROM animals
+      WHERE name = 'Agumon'
+    ),
+    (
+      SELECT id
+      FROM vets
+      WHERE name = 'William Tatcher'
+    ),
+    '05/24/2020'
+  ),
+  (
+    (
+      SELECT id
+      FROM animals
+      WHERE name = 'Agumon'
+    ),
+    (
+      SELECT id
+      FROM vets
+      WHERE name = 'Stephanie Mendez'
+    ),
+    '07/22/2020'
+  ),
+  (
+    (
+      SELECT id
+      FROM animals
+      WHERE name = 'Gabumon'
+    ),
+    (
+      SELECT id
+      FROM vets
+      WHERE name = 'Jack Harkness'
+    ),
+    '02/02/2021'
+  ),
+  (
+    (
+      SELECT id
+      FROM animals
+      WHERE name = 'Pikachu'
+    ),
+    (
+      SELECT id
+      FROM vets
+      WHERE name = 'Maisy Smith'
+    ),
+    '01/05/2020'
+  ),
+  (
+    (
+      SELECT id
+      FROM animals
+      WHERE name = 'Pikachu'
+    ),
+    (
+      SELECT id
+      FROM vets
+      WHERE name = 'Maisy Smith'
+    ),
+    '03/08/2020'
+  ),
+  (
+    (
+      SELECT id
+      FROM animals
+      WHERE name = 'Pikachu'
+    ),
+    (
+      SELECT id
+      FROM vets
+      WHERE name = 'Maisy Smith'
+    ),
+    '05/14/2020'
+  ),
+  (
+    (
+      SELECT id
+      FROM animals
+      WHERE name = 'Devimon'
+    ),
+    (
+      SELECT id
+      FROM vets
+      WHERE name = 'Stephanie Mendez'
+    ),
+    '05/04/2021'
+  ),
+  (
+    (
+      SELECT id
+      FROM animals
+      WHERE name = 'Charmander'
+    ),
+    (
+      SELECT id
+      FROM vets
+      WHERE name = 'Jack Harkness'
+    ),
+    '02/24/2021'
+  ),
+  (
+    (
+      SELECT id
+      FROM animals
+      WHERE name = 'Squirtle'
+    ),
+    (
+      SELECT id
+      FROM vets
+      WHERE name = 'Stephanie Mendez'
+    ),
+    '09/29/2019'
+  ),
```

Comment:
- [ ] There should be a total of 20 insertions. In your case, you only did 17. I believe you forgot to add animal `Plantmon` vets. They should be added between animal `Charmander` and `Squirtle`